### PR TITLE
Add first tests for OpenShift use cases

### PR DESCRIPTION
**Pull request by hhorak**

### CODE REVIEW

**Comment by hhorak** [comment]
[test-openshift]

File: test/run-openshift [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+# Test the httpd image in OpenShift.
+#
+# IMAGE_NAME specifies a name of the candidate image used for testing.
+# VERSION specifies a version of the python in the candidate image.
+# The image has to be available before this script is executed.
+
+THISDIR=$(dirname ${BASH_SOURCE[0]})
+
+source ${THISDIR}/test-lib.sh
+source ${THISDIR}/test-lib-openshift.sh
```
Comment (pkubatrh) [inline test/run-openshift]: These need to be double quoted.  Also `IMAGE_NAME` needs double quotes added when the tests are invoked.

File: test/run-openshift [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+# TODO: branch should be changed to master, once code in example app
```
Comment (pkubatrh) [inline test/run-openshift]: Since BRANCH_TO_TEST is set to master shouldn't this get removed?

Comment (hhorak) [reply test/run-openshift]: Ack, will do.

File: test/run-openshift (hunk 1 above)
Comment (hhorak) [reply test/run-openshift]: Good point, will fix.

**Comment by hhorak** [comment]
[test-openshift]

**Comment by pkubatrh** [comment]
lgtm, merging.

**Pull request merged**